cypress/integration/checkout/checkout.basic.spec.js

```diff
@@ -1,54 +1,122 @@
+import uuid from 'uuid';
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import CheckInPane from '../../support/fragments/check-in-actions/checkInPane';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
 import generateItemBarcode from '../../support/utils/generateItemBarcode';
-import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+import getRandomPostfix from '../../support/utils/stringTools';
+import Users from '../../support/fragments/users/users';
+import devTeams from '../../support/dictionary/devTeams';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
 
-describe('Check Out', () => {
-  const ITEM_BARCODE = generateItemBarcode();
-  let source;
+describe('Check Out - Actions ', () => {
+  const userData = {
+    permissions: [
+      permissions.checkinAll.gui,
+      permissions.uiCirculationViewCreateEditDelete.gui,
+      permissions.uiUserCreate.gui,
+      permissions.inventoryAll.gui,
+      permissions.checkoutCirculatingItems.gui,
+    ],
```

Comment:
Please move permissions to the place where you use it one time to the parameter of the method  cy.createTempUser